validation/validation.go

```diff
@@ -1,27 +1,41 @@
 package validation
 
 import (
+	"errors"
 	"fmt"
 	"net/url"
 
 	"github.com/fatih/color"
 	"github.com/preslavmihaylov/todocheck/config"
+	"github.com/preslavmihaylov/todocheck/fetcher"
+	"github.com/preslavmihaylov/todocheck/issuetracker"
 )
 
 // Validate validates the values of given configuration
 func Validate(cfg *config.Local) []error {
-	var errors []error
+	var errs []error
 
 	if err := validateIssueTracker(cfg); err != nil {
-		errors = append(errors, err)
+		errs = append(errs, err)
 	}
 
 	if err := validateAuthOfflineURL(cfg); err != nil {
-		errors = append(errors, err)
+		errs = append(errs, err)
 	}
 
 	if err := validateIssueTrackerOrigin(cfg); err != nil {
-		errors = append(errors, err)
+		errs = append(errs, err)
+	}
+
+	url, err := issuetracker.HealthcheckURL(cfg.IssueTracker, cfg.Origin)
+	if err != nil {
+		if !errors.Is(err, issuetracker.ErrUnsupportedHealthCheck) {
+			errs = append(errs, err)
+		}
+	} else {
+		if !fetcher.IsHealthy(cfg.IssueTracker, url) {
```

Comment:
This can be simplified from:
```
} else {
    if !fetcher.IsHealthy(cfg.IssueTracker, url) {
```

to
```
} else if !fetcher.IsHealthy(cfg.IssueTracker, url) {
```

Reply:
I ended up simplifying this flow a bit after extracting to an unexported function. I think it reads better but let me know what you think!